ROADMAP.md

```diff
@@ -55,6 +55,22 @@ Please file an issue to make suggestions on this roadmap!
       * Dynamic reconfiguration of cluster-autoscaler
       * Automatic discovery of target node pools for cluster-autoscaler
     * Requires much work on CA side
+
+## v0.9.8
+
+  * Experimental support for kube-aws plugins
```

Comment:
Accordingly to #509 
I'm looking forward to @c-knowles's design proposal for this :bow: